### PR TITLE
2024-02-13-bert_sequence_classifier_biomarker_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+engine: tensorflow
+annotator: MedicalBertForSequenceClassification
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model is a [BioBERT](https://nlp.johnsnowlabs.com/2022/07/18/biobert_pubmed_base_cased_v1.2_en_3_0.html) based sentence classification model that can determine whether the clinical sentences include terms related to biomarkers or not.
```
Comment (muhammetsnts) [inline docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md:22]: Is this link correct? You didn't use this embeddings as far as I know.

File: docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+  
+```python
+document_assembler = DocumentAssembler() \
+    .setInputCol('text') \
+    .setOutputCol('document')
+
+sentence_detector = SentenceDetector() \
```
Comment (muhammetsnts) [inline docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md:48]: Use clinical sentence detector in all clinical model samples.

File: docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+model = pipeline.fit(data)
+result = model.transform(data)
+```
+```scala
+val documentAssembler = new DocumentAssembler()
+  .setInputCol("text")
+  .setOutputCol("document")
+
+val sentenceDetector = new SentenceDetector()
```
Comment (muhammetsnts) [inline docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md:77]: Use clinical sentence detector in all clinical model samples.

File: docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+val model = pipeline.fit(data)
+val result = model.transform(data)
+
+```
+</div>
+
+## Results
+
+```bash
++----------+------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (muhammetsnts) [inline docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md:109]: Move prediction column to right side.

**Review by muhammetsnts — CHANGES_REQUESTED** [review]
Please fix here.

File: docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md (hunk 1 above)
Comment (ahmet-mesut) [reply docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md]: fixed FYI

File: docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md (hunk 2 above)
Comment (ahmet-mesut) [reply docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md]: changed with sentence detectordl helathcare model

**Pull request merged**